src/languages/_french.json

```diff
@@ -1437,7 +1437,8 @@
       "hash": "Tx Hash",
       "network_fee": "Network Fee",
       "hash_copied": "\uDBC0\uDC63 Transaction Hash Copied",
-      "address_copied": "\uDBC0\uDC63 Address Copied"
+      "address_copied": "\uDBC0\uDC63 Address Copied",
```

Comment:
for the strings that aren't translated in this file (still english), we attach a ":)" at the end of the string (can see the other strings in this file for example)